airflow/__init__.py

```diff
@@ -37,7 +37,7 @@
 
 __version__ = version.version
 
-__all__ = ['__version__', 'login', 'DAG', 'PY36', 'PY37', 'PY38', 'PY39', 'PY310']
+__all__ = ['__version__', 'login', 'DAG', 'PY36', 'PY37', 'PY38', 'PY39', 'PY310', 'XComArg']
```

Comment:
Is this worth adding to `__all__`?

Reply:
Yes, I meant to.

Reply:
Oh Sorry, I See. Dunno. 😁

Reply:
Given that `DAG` is, perhaps? (Also why is `AirflowException` not in this list?)